app/extensions/string.rb

```diff
@@ -739,10 +739,6 @@ def md5sum
 
   ### Misc Utilities ###
   #
-  def print_thing(thing)
```

Comment:
`print_thing` isn't called anywhere.

Reply:
Seems useful (and intended) for debugging, so it wouldn't stay in the code base. But i didn't know about it.